datasource/src/main/resources/liquibase/changelog/changelog-0.1.xml

```diff
@@ -32,4 +32,13 @@
         </rollback>
     </changeSet>
 
+    <changeSet id="add_column_for_user_deactivation" author="ysheiko">
+        <addColumn tableName="user">
+            <column name="active" type="boolean" defaultValueBoolean="true"/>
```

Comment:
I think it's worth adding boolean type as a property in liquibase (properties.xml)

Reply:
Additional property was created